keyboards/lets_split/keymaps/DE_programming/keymap.c

```diff
@@ -0,0 +1,153 @@
+#include QMK_KEYBOARD_H
+#include "keymap_german.h"
+
+extern keymap_config_t keymap_config;
```

Comment:
Unused
```suggestion
```

Reply:
Not just unused, it's included already, so no need to have it. It's redundant.